### ISSUE TITLE
Mojs Player Latency

### ISSUE THREAD

**Issue by legomushroom** [mojs/mojs-player#3]
@zfrisch commented on [Wed Dec 21 2016](https://github.com/legomushroom/mojs/issues/93)

Hello! Strangely I'm having an issue with the Mojs Player tracker line being latent. It's only an issue in my Chrome Browser(Version 55.0.2883.87 m, which is the most current). In FF and IE10/Edge it seems to work fine. The error also doesn't seem to show up on CrossBrowserTesting (the site that CodePen uses for Cross Browser Testing). 

I recorded a video to show what I'm talking about: 
https://www.youtube.com/watch?v=K_ib-3OWCuE 

Here's the CodePen : 
http://codepen.io/zfrisch/full/yVwmoW/

Thanks!

---

@eltonmesquita commented on [Wed Dec 21 2016](https://github.com/legomushroom/mojs/issues/93#issuecomment-268596648)

Hello @zfrisch . There's a big chance your problem is related to a chrome extension. It's a really common issue. Try to run it in an anonymous window and see if the issue persists.

---

@zfrisch commented on [Wed Dec 21 2016](https://github.com/legomushroom/mojs/issues/93#issuecomment-268598961)

Hi @eltonmesquita ! Thanks for the advice. I tried this in Incognito Mode and made sure my Chrome extensions were all disabled for good measure. Unfortunately it's still showing the same on my screen.

---

@legomushroom commented on [Wed Dec 21 2016](https://github.com/legomushroom/mojs/issues/93#issuecomment-268614844)

Hi Zak, Hi Elton!

I have seen that issue when you use the player inside an iFrame, for instance it behaves like that when you use it with webpack-dev-server. Is it true for you? Any environments details I should know about? 

Thanks a lot for the issue!

---

@legomushroom commented on [Wed Dec 21 2016](https://github.com/legomushroom/mojs/issues/93#issuecomment-268615138)

Oh yeah, I think we should move this issue to https://github.com/legomushroom/mojs-player repo

---

@legomushroom commented on [Wed Dec 21 2016](https://github.com/legomushroom/mojs/issues/93#issuecomment-268615663)

Oh I see, that's another issue actually. Interesting. Never saw that. That's for the video and codepen sample.

**Comment by legomushroom** [comment]
@zfrisch @eltonmesquita

**Comment by legomushroom** [comment]
Finally reproduced this issue with Chrome Version 55.0.2883.95 (64-bit) will try to fix it asap for you. The issue popups when you have some page zoom level that is different from `100%`.

**Comment by zfrisch** [comment]
@legomushroom the percentage of zoom actually does seem to change it. On the previous video the zoom  is the default of 100%, but when I zoom out to 80% it works perfectly.

New video:
https://www.youtube.com/watch?v=WDUDtnjFWPM&feature=youtu.be&hd=1

**Comment by legomushroom** [comment]
Hm so you have it without any zoom, that's weird. I know what will solve it but it might be less performant though.

**Comment by zfrisch** [comment]
@legomushroom That's interesting! To be honest it's not a deal-breaker or anything, for me at least, and if it hinders performance to correct, it may not be worth it.  A temporary solution may be to just hide the progress overlay on the tracker for this browser? The tracker handle works just fine and it's probably enough!

**Comment by legomushroom** [comment]
@zfrisch the problem is that the issue doesn't show up for me on the same browser so it is hard to target it. Trying to nail down the issue. Merry Christmas! :)

**Comment by legomushroom** [comment]
@zfrisch made a lot of experiments and didn't find the solution without performance overhead since this is the tool to debug animations, I'd keep it as simple as possible. Could you tell me more your machine? Operating system/Screen density/Device. Thanks a lot!

**Comment by zfrisch** [comment]
Sure! I'd be happy to provide that info. I will when i get a chance! I was
thinking it may have been due to experimental features being on? I'll take
a look under the hood when i get home and let you know. Thanks for looking
into it Oleg and merry Christmas!

**Comment by legomushroom** [comment]
@zfrisch Merry Christmas! I've pushed the `0.43.16` version that might solve the issue only for `zoom: 100%`. Please check it out.

**Comment by zfrisch** [comment]
Hi Oleg! The new version of mojs-player unfortunately didn't fix it, however I did figure out what was happening!

The bug is created because of a feature introduced in Chrome's newer versions. 

> "With the Chrome 54 update ... Chrome can detect the DPI setting on your computer so that the web browser will be scaled accordingly." 

I found my solution here: http://blog.the-ebook-reader.com/2016/10/21/how-to-fix-google-chromes-large-zoomed-in-ui-after-update/ - and everything seems to be working as it should!

Thanks for all your help!
